package.json

```diff
@@ -28,8 +28,9 @@
   "system": {},
   "dependencies": {
     "escape-html": "^1.0.3",
-    "unescape-html": "^1.0.0",
-    "jquery": "^2.2.4"
+    "jquery": "^2.2.4",
+    "lodash": "^4.17.4",
```

Comment:
If it’s deemed overkill to add lodash as a dependency, it can be removed, I just didn’t want to write my own debounce function unless I needed to. Note that only `lodash/debounce` is being required so I don’t think it will bring in all of lodash.